src/app/components/Figure/Image/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+
+const StyledImage = styled.img`
+  display: block;
+  width: 100%;
+`;
+
+const Image = ({ src, alt }) => <StyledImage alt={alt} src={src} />;
```

Comment:
This shouldn't be a component that wraps the styled component, just export the `styled.img`.

Reply:
How can I add the prop types to ensure an `alt` and `src` is passed if this just exports `styled.img`?

Reply:
You don't need to add prop types. A styled component (`styled.img`) is just a react component wrapping a DOM element and applying styles that are passed in a template literal block. The prop types passed to the styled component are passed as attributes to the DOM element.

Reply:
Forgot the prop types were now being handled in the Figure container, my mistake.